content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -3188,105 +3188,242 @@ calendar.
+   :noindex:
+
+   If true, open events in dialog to edit them, otherwise, open them in a
+   classical form view.
+
+   :requirement: Optional
+   :type: bool
+   :default: `False`
+
+.. attribute:: show_date_picker
+   :noindex:
+
+   By default, a mini calendar (in month mode) is displayed in the side panel,
+   next to the main calendar. This option allows to remove it by setting it to
+   `False`.
+
+   :requirement: Optional
+   :type: bool
+   :default: `True`
+
+.. attribute:: form_view_id
+   :noindex:
+
+   View to open when the user creates or edits an event. By default, uses the
+   form view of the current action, if any.
+
+   :requirement: Optional
+   :type: int
+
+.. attribute:: quick_create
+   :noindex:
+
+   Enables quick event creation on click: only asks the user for a ``name`` and
+   tries to create a new event with just that and the clicked event time. Falls
+   back to a full form view if the quick creation fails.
+
+   :requirement: Optional
+   :type: bool
+   :default: `True`
+
+.. attribute:: create_name_field
+   :noindex:
+
+   Name of the record's field holding the display name of the record. This field
+   is used when creating records through the 'quick create' mechanism.
 
-  ``invisible``
-    use "True" to hide the value in the cards
-  ``avatar_field``
-    only for x2many field, to display the avatar instead of the display_name
-    in the cards
-  ``write_model`` and ``write_field`` and ``filter_field``
-    you can add a filter and save the result in the defined model, the
-    filter is added in the sidebar. The ``filter_field`` is optional and allows
-    you to specify the field that will hold the status of the filter.
-  ``filters`` and ``color``
-    use "True" to add this field in filter in the sidebar. You can specify
-    a ``color`` field used to colorize the checkbox.
-
-
-Model Commons
--------------
-
-.. currentmodule:: odoo.addons.base.models.ir_ui_view
-.. autoattribute:: Base._date_name
-    :noindex:
+   :requirement: Optional
+   :type: str
+   :default: `name`
+
+.. attribute:: quick_create_view_id
+   :noindex:
+
+   Id of the form view to open when the attribute ``quick_create`` is set and the
+   user creates an event, instead of the default dialog which only allows to
+   specify a name.
+
+   :requirement: Optional
+   :type: int
+
+.. include:: view_architectures/root_attribute_create.rst
+
+.. include:: view_architectures/root_attribute_edit.rst
+
+.. attribute:: delete
+   :noindex:
+
+   Disable/enable record deletion on the view.
+
+   :requirement: Optional
+   :type: bool
+   :default: `True`
+
+.. include:: view_architectures/root_attribute_string.rst
+
+.. _reference/view_architectures/calendar/components:
+
+Components
+----------
+
+Calendar views accept a single type of child elements: ``<field>``. Those fields
+are displayed in a popover, in the given order, which opens when a record
+(a calendar event) is clicked. Field nodes can have the following attributes:
```

Comment:
We need to note that the fields displayed in the popover are read only. They are not editable.